pkg/distributor/distributor.go

```diff
@@ -626,9 +629,7 @@ func (d *Distributor) validateSeries(nowt time.Time, ts *mimirpb.PreallocTimeser
 	for _, s := range ts.Samples {
 
 		delta := now - model.Time(s.TimestampMs)
-		if delta > 0 {
-			d.sampleDelayHistogram.Observe(float64(delta) / 1000)
-		}
+		d.sampleDelayHistogram.Observe(float64(delta) / 1000)
```

Comment:
I'm wondering how expensive it is to call `Observe()` for every single sample. Do you have a benchmark we can run, to compare it before vs after this change?

Reply:
You're totally right. I initially read the code the other way. I'm good, thanks.